loader/loader_json.c

```diff
@@ -131,20 +147,22 @@ static VkResult loader_read_entire_file(const struct loader_instance *inst, cons
 }
 #else
 #warning fopen not available on this platform
-VkResult loader_read_entire_file(const struct loader_instance *inst, const char *filename, size_t *out_len, char **out_buff) {
+VkResult loader_read_entire_file(const struct loader_instance *inst, int parent_dir_fd, const char *filename, size_t *out_len,
+                                 char **out_buff) {
     return VK_ERROR_INITIALIZATION_FAILED;
 }
 #endif
 
-TEST_FUNCTION_EXPORT VkResult loader_get_json(const struct loader_instance *inst, const char *filename, cJSON **json) {
+TEST_FUNCTION_EXPORT VkResult loader_get_json(const struct loader_instance *inst, int parent_dir_fd, const char *filename,
```

Comment:
I mentioned my questions about int parent_dir_fd, but I almost forgot to mention how changing this functions signature will break the OSS-Fuzz Vulkan-Loader job as it directly calls this (and a few other) functions. 

I should probably have documented that in the description of each function, so that's on me more than anything.